perllib/FixMyStreet/Roles/ContactExtra.pm

```diff
@@ -0,0 +1,49 @@
+package FixMyStreet::Roles::ContactExtra;
+
+use Moo::Role;
+use JSON::MaybeXS;
+
+requires 'join_table', 'map_extras';
+
+sub fetch_bodies {
```

Comment:
Was the idea to use this function in by_categories? It's currently not being used.